packages/slider/src/HandleController.ts

```diff
@@ -432,7 +445,15 @@ export class HandleController {
         this.requestUpdate();
     };
 
-    private onInputKeydown = (event: Event): void => {
+    private onInputKeydown = (event: KeyboardEvent): void => {
+        if (event.key == 'Enter') {
```

Comment:
The `Enter` key, when pressed while focusing a form field, should trigger the `change` event on that form field, not reset it. The accessibility team spoke of possibly accepting `Escape` as a resetting key, but we should confirm with this this behavior and any associated messaging that should be surfaced by the element to inform screen reader users that this functionality is available. Further, we should then confirm how the use of `Escape` in that context may or may not interfere with pressing `Escape` while in a modal, at which point the `Escape` key would be expected to close the modal...

Reply:
Thank you for this feedback... Super helpful. I will switch the `Enter` key with `Escape`.
1. For the accessibility part I'm thinking if we can have an `aria-describedby` on the input that points to the description of this specific functionality?
2. And I verified the current code in case of an modal and currently on pressing `escape` the modal does close however the value also resets to default. Is this expected or do we want to not reset the value and just close the modal if the slider is in a modal?